.github/ISSUE_TEMPLATE/crash-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Crash report
+about: Report a game crash.
+title: My game crashed
+labels: Crash
+assignees: ''
+
+---
+
+<!-- This is a guideline that shall help you to include all the required information we depend on to investigate and fix game-breaking bugs. Please follow the instructions and replace any placeholders that are written in capital letters. Instructions like this comment will not be visible in your report. -->
+
+## System Information
+<!-- Information about the operating system, engine version, game mod and package source are mandatory for investigating and fixing crashes. -->
+
+- Operating System:  OPERATING SYSTEM
+- OpenRA Version: ENGINE VERSION
+- OpenRA Mod: GAME MOD
+- Source: Official download package OR self-compiled OR third-party package
+
```

Comment:
I removed .NET / Mono Version from here because on Windows it is really hard to find the correct information if you aren't familiar with windows-programming. Since the Mono runtime is included in the packages, I thought this information is no longer needed here.

Reply:
Mono version is still important for self-compiled and third-party packages.